js-es2015-react.md

```diff
@@ -0,0 +1,126 @@
+# Tooling
+
+For projects that use ES2015, we use:
+
+- eslint
```

Comment:
Would be good to link those up:
- http://eslint.org/
- https://github.com/yannickcr/eslint-plugin-react

So newcomers/people can read up straight away